### DISCUSSION TITLE
Rebuild database

### DISCUSSION THREAD

**Discussion by oguime** [blakeblackshear/frigate#14337] [Ask A Question] [answered]
I want to redeploy Frigate mapping the **config** and **media** folders to different paths but want to keep the current clips, exports and recordings available from the UI.

Is there a way to move the existing media files to the new location and rebuild the database?

Thanks!

**Comment by NickM-27** [comment, marked as answer]
the paths inside of the docker container should never change, so you can change the paths that are on the host to whatever you want and it doesn't matter as long as the docker mapping stays consistent

**Reply by oguime** [reply]
Got it! Just copied the database and media files to the new folders keeping the docker paths.
Thanks!
